src/desktop/sudo.desktop

```diff
@@ -13,6 +13,7 @@ X-SuSE-YaST-AutoInst=
 X-SuSE-YaST-Geometry=
 X-SuSE-YaST-SortKey=
 X-SuSE-YaST-AutoInstResource=sudo
+X-SuSE-YaST-Keywords=root,permissions
```

Comment:
Maybe also `users`, `authentication` and definitely `administrator` (`root` might not be known by beginners).